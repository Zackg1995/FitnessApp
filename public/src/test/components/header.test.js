import ReactShallowRenderer from "react-test-renderer/shallow";
import React from "react";
import Header from "../../components/header";

test("Should render the header", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Header />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();

  console.log(renderer.getRenderOutput());
});
