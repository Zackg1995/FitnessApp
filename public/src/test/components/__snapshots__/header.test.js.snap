// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the header 1`] = `
<div
  className="header"
>
  <div
    className="headerContainer"
  >
    <header>
      <h1
        className="header_title"
      >
        FitBud
      </h1>
      <NavLink
        activeClassName="is-active"
        className="nav"
        exact={true}
        to="/"
      >
        Home Page
      </NavLink>
      <NavLink
        activeClassName="is-active"
        className="nav"
        to="/create"
      >
        Adding Items Page
      </NavLink>
      <NavLink
        activeClassName="is-active"
        className="nav"
        to="/help"
      >
        Help Page
      </NavLink>
    </header>
  </div>
</div>
`;
